plugins/isplink-client-registration/src/public.php

```diff
@@ -0,0 +1,225 @@
+?>
+<!DOCTYPE html>
+<html>
+<head>
+    <title><?php echo htmlspecialchars($config['PAGE_TITLE'] ?? 'Sign Up for Service'); ?></title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap/5.3.0/css/bootstrap.min.css" rel="stylesheet">
+    <link href="<?php echo $publicUrl . 'styles.css'?>" rel="stylesheet">
+</head>
+<body>
+    <!-- Hero Section with Logo in Top Left -->
+    <div class="simple-hero">
+    <div class="hero-container">
+        <!-- Logo positioned in top left -->
+        <div class="hero-logo mb-5">
+            <?php if (!empty($config['LOGO_URL'])): ?>
+                <img src="<?php echo htmlspecialchars($config['LOGO_URL']); ?>" alt="<?php echo htmlspecialchars($config['COMPANY_NAME'] ?? 'Company Logo'); ?>">
+            <?php elseif (!empty($config['COMPANY_NAME'])): ?>
+                <h1><?php echo htmlspecialchars($config['COMPANY_NAME']); ?></h1>
+            <?php else: ?>
+                <img src="https://isplink.app/img/isplink_logo.png" alt="Default Logo">
+            <?php endif; ?>
+        </div>
+        
+        <div class="hero-content">
+            <h1><?php echo htmlspecialchars($config['HEADING'] ?? ''); ?></h1>
+            <p><?php echo htmlspecialchars($config['SUBHEADING'] ?? ''); ?></p>
+        </div>
+    </div>
+</div>
+
+    <!-- Stepper Navigation -->
+    <div class="stepper-container mt-2">
+        <div class="simplified-stepper">
+            <div class="step active">
+                <div class="step-circle">1</div>
+                <div class="step-label">Availability</div>
+            </div>
+            
+            <div class="step">
+                <div class="step-circle">2</div>
+                <div class="step-label">Service Plan</div>
+            </div>
+            
+            <div class="step">
+                <div class="step-circle">3</div>
+                <div class="step-label">Account</div>
+            </div>
+            
+            <div class="step">
+                <div class="step-circle">4</div>
+                <div class="step-label">Installation</div>
+            </div>
+            
+            <div class="step">
+                <div class="step-circle">5</div>
+                <div class="step-label">Review</div>
+            </div>
+            
+            <div class="step">
+                <div class="step-circle">6</div>
+                <div class="step-label">Confirmation</div>
+            </div>
+        </div>
+    </div>
+
+    <!-- Main Content -->
+    <div class="container py-4">
+        <div class="row justify-content-center">
+            <div class="col-sm-10 col-md-8 col-lg-6">
+                <!-- API Error Alert (if there was an API error) -->
+                <?php if ($apiError): ?>
+                    <div class="alert alert-danger mb-4">
+                        <?php echo $apiError; ?>
```

Comment:
XSS possible, I see the link on L48, but you'll have to handle it differently, we can't allow direct write into HTML from a 3rd party response in the other cases. Escaping it there would be fine.
```suggestion
                        <?php echo htmlspecialchars($apiError, ENT_QUOTES); ?>
```